lib/Extractor/Candidates.cpp

```diff
@@ -171,7 +181,26 @@ Inst *ExprBuilder::makeArrayRead(Value *V) {
       Negative = isKnownNegative(V, DL);
       NumSignBits = ComputeNumSignBits(V, DL);
     }
-  return IC.createVar(Width, Name, Known.Zero, Known.One, NonZero, NonNegative,
+
+    // constant range
+    ConstantRange Range = llvm::ConstantRange(Width, true);
```

Comment:
for boolean function arguments always include a comment like this "ConstantRange(Width, /*isFullSet=*/false);"